### DISCUSSION TITLE
Question about https://s.mble.dk

### DISCUSSION THREAD

**Discussion by mbledkowski** [searxng/searx-instances#611] [General]
Hello,
My instance was down for about 3 days, due to the fact, that I forgot to pay for my domain.
Everything now works as it should.
My instance is currently being listed in the Offline & Error category.
I have no idea on whether it is an automatic process, or a manual one; but I can assure you, that the instance is ready to be added back to the online category

**Comment by ononoki1** [comment]
Hi @mbledkowski, your instance is up but it's very slow and sometimes returns 502 error. Please check your machine.

**Reply by mbledkowski** [reply]
![image](https://github.com/user-attachments/assets/7ab04486-f26f-4509-b606-0b78e528834f)
My instance was being hammered at the time being; no idea what happened here.
